themes/bootstrap3/templates/myresearch/mylist.phtml

```diff
@@ -16,6 +16,7 @@
   if ($this->params->getOptions()->getListViewOption() != "full") {
     $this->headScript()->appendFile("record.js");
     $this->headScript()->appendFile("embedded_record.js");
+    $this->headScript()->appendFile("truncate.js");
```

Comment:
@aleksip, is it intentional that you are only loading truncate.js when the list view option is not full (that is, when the option for ajax-loading records into a tabbed or accordioned control is active, instead of linking to the record page)? If I'm understanding correctly, I think we're more likely to ALWAYS want the truncate.js to be available, but maybe I'm mistaken.

Reply:
@demiankatz To be honest, I was not at all sure about this approach. I just added `truncate.js` wherever `record.js` was added. So should I just remove all these and add  `$this->headScript()->appendFile("truncate.js");` to `layout/layout.phtml` instead?

Reply:
@aleksip, if we're going to be using the truncation feature globally, the easiest thing is probably to add it to the list of Javascript files in the theme configuration here: https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap3/theme.config.php#L44

Reply:
@demiankatz Ah, yes, of course! Have now done that in uchicago-library#112 and removed the draft status. @bbusenius